pilot/lib/pilot/build_manager.rb

```diff
@@ -49,6 +51,8 @@ def upload(options)
         return
       end
 
+      login_if_not_logged_in
```

Comment:
Code style nitpick: methods with no input and no outputs give me pause. Why does this exist if it's only called once (and it doesn't appear to be used in mocking or anything like that) ?

Reply:
@joshdholtz, your code.

Reply:
ping @joshdholtz